added call out component to intro page (#174)
usebruno/bruno-docs: src/pages/git-integration/using-gui/intro.mdx

import PremiumBadge from "@/components/premium-badge";
import { Callout } from 'nextra/components'

# **Collaboration via GUI** <PremiumBadge />

Bruno offers an intuitive **GUI-based Git collaboration** feature, available in the [Pro](https://www.usebruno.com/buy?plan=pro) and [Ultimate Editions](https://www.usebruno.com/buy?plan=ultimate). This feature makes it easier for both engineering and non-engineering teams to collaborate seamlessly on collections.

<Callout type="warning" emoji="⚠️">

> You need to have Git installed on your machine to use these feature.

Reviewer: ```suggestion
Git must be [installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your machine to use this feature. 
```
